locales/sq_AL/LC_MESSAGES/duckduckgo.po

```diff
@@ -818,13 +820,13 @@ msgstr "Klikoni %sKëtu%s për të shkarkuar shtesën e DuckDuckGo"
 msgid ""
 "Click %sOK%s in the popup. %sNOTE%s: If OK button is disabled, it was "
 "already added. Click %sCancel%s and proceed to the next step."
-msgstr ""
+msgstr "Kliko %sOK%s në dritaren e shfaqur. %sVINI RE%s: Nëse butoni OK është i çaktivizuar "
+"dmth ka qenë veçmë i shtuar. Kliko mbi %sAnuo%s dhe vazhdo më tej."
 
 msgid "Click %sOpen%s to download and open the DuckDuckGo Safari extension"
 msgstr ""
 "Klikoni %sOpen%s (Hap) për të shkarkuar dhe hapur shtesën DuckDuckGo për "
 "Safari"
-
```

Comment:
Please, add back this white space, otherwise the software will detect a syntax error.

Reply:
@GioSensation : not sure I undersstood.  :eyes:

Reply:
Sorry for not being clear. There should be an empty line here, but it was removed. This should be the cause for the CI checks failing. If you can please add it back, we should be good to go.

Reply:
@GioSensation  NP.  Also first time i'm doing this. Should the white space come as an entire line between 827 and 828, or I should add a white space within 827 somewhere? I'm not sure where, i apologise.

Reply:
Not quite. Translations are organised as blocks, you usually have this format:

```
msgctxt "Here we add context for you to understand what you are translating (this is optional)"
msgid "This is the string to translate"
msgstr "And here goes the actual translation"
```

The order must be respected and also blocks should be separated by new lines. I have pushed an update myself, but it looks like there is some other problem. I'll look into it and let you know.

Also, no need to apologies, and thanks again for your efforts. Much appreciated.

Reply:
> Not quite. Translations are organised as blocks, you usually have this format:
> 
> ```
> msgctxt "Here we add context for you to understand what you are translating (this is optional)"
> msgid "This is the string to translate"
> msgstr "And here goes the actual translation"
> ```
> 
> The order must be respected and also blocks should be separated by new lines. I have pushed an update myself, but it looks like there is some other problem. I'll look into it and let you know.
> 
> Also, no need to apologies, and thanks again for your efforts. Much appreciated.

All righty. Let me know for anything.